accelerator/roles/intel/tasks/install_ubuntu.yml

```diff
@@ -0,0 +1,119 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Check if accelerator is present on node
+  ansible.builtin.include_tasks: verify_has_accelerators.yml
+
+- name: Include accelerator_config
+  ansible.builtin.include_vars: "{{ role_path }}/../../../input/accelerator_config.yml"
+
+- name: Common setups
+  block:
+    - name: Gather package facts
+      ansible.builtin.package_facts:
+        manager: auto
+
+    - name: Check if kernel supported
+      ansible.builtin.fail:
+        msg: "Kernel not supported"
+      when: ansible_kernel is version('5.4.0', '<')
+
+    # TODO: check if this is the right place to do this.
+    - name: Base dependencies  # noqa package-latest
+      ansible.builtin.apt:
+        name: "{{ apt_base_packages | list }}"
+        state: latest
+        update_cache: true
+
+    # TODO: check with Dell first (this might have side effects)
+    # required by kubeflow: https://github.com/kubeflow/manifests/issues/2087
+    - name: Increase fs.inotify.max_user_instances
+      ansible.posix.sysctl:
+        name: fs.inotify.max_user_instances
+        value: '1280'
+        state: present
+        reload: true
+        sysctl_set: true
+        ignoreerrors: false
+        sysctl_file: /etc/sysctl.d/99-sysctl.conf
+
+- name: Setup compute nodes
+  when: node_has_accelerator
+  block:
+    # TODO: need to move Tuned outside of the Gaudi installation playbook
+    - name: Start Tuned service
+      ansible.builtin.systemd_service:
```

Comment:
there should be separate utility for tuneD profile configuration

Reply:
I agree we just need to track this requirement